tests/tensor/test_nlinalg.py

```diff
@@ -606,12 +606,10 @@ def test_perform(self):
                 np.testing.assert_allclose(out, np_val)
 
     def test_kron_commutes_with_inv(self):
-        for shp0, shp1 in zip(
-            [(2, 3), (2, 3), (2, 4, 3)], [(6, 7), (4, 3, 5), (4, 3, 5)]
+        for i, (shp0, shp1) in enumerate(
+            zip([(2, 3), (2, 3), (2, 4, 3)], [(6, 7), (4, 3, 5), (4, 3, 5)])
         ):
-            if (pytensor.config.floatX == "float32") & (
-                len(shp0) == 3 or len(shp1) == 3
-            ):
+            if (pytensor.config.floatX == "float32") & (i == 2):
                 pytest.skip("Kron and inv do not commute at half precision")
```

Comment:
Last nitpik, the issue is actually with `pinv` I think, just write as the comment "Half precision insufficient for test 3 to pass"

Reply:
ah yes i wasnt sure about the correctness of my comment. i'll update it